nixos/modules/programs/regreet.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+let
+  cfg = config.programs.regreet;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.programs.regreet = {
+    enable = lib.mkEnableOption null // {
+      description = lib.mdDoc ''
+        Enable ReGreet, a clean and customizable greeter for greetd.
+
+        To use ReGreet, {option}`services.greetd` has to be enabled and
+        {option}`services.greetd.settings.default_session` should contain the
+        appropriate configuration to launch
+        {option}`config.programs.regreet.package`. For examples, see the
+        [ReGreet Readme](https://github.com/rharish101/ReGreet#set-as-default-session).
+      '';
+    };
+
+    package = lib.mkPackageOptionMD pkgs "regreet" { };
```

```suggestion
    package = lib.mkPackageOptionMD pkgs [ "greetd" "regreet" ] { };
```

This is only available as `pkgs.greetd.regreet`, right, not as `pkgs.regreet`?

---

Right.

---

Yeah then it needs that diff to eval